include/util/hex_float.h

```diff
@@ -735,6 +772,8 @@ inline bool RejectParseDueToLeadingSign(std::istream& is, bool negate_value,
 // If negate_value is true then the number may not have a leading minus or
 // plus, and if it successfully parses, then the number is negated before
 // being stored into the value parameter.
+// If the value is an infinity, then set the fail bit on the stream, and set
```

Comment:
Good point.  Actually, the comment is not quite right.  If it fails to parse as a number, the value is set to 0.  Will fix in followup commit.